tools/snap/generate_dnsplugins_snapcraft.sh

```diff
@@ -35,7 +35,15 @@ parts:
       - PIP_CONSTRAINT: \$SNAPCRAFT_PART_SRC/snap-constraints.txt
       - SNAP_BUILD: "True"
     # To build cryptography and cffi if needed
-    build-packages: [gcc, libffi-dev, libssl-dev, python3-dev]
+    build-packages:
+      - gcc
+      - git
+      - libaugeas-dev
```

Comment:
nit: I don't think we need Augeas in our DNS plugins.

Reply:
lollll yes I did just copy and paste from the certbot snapcraft file

Reply:
Makes sense. I almost didn't notice it either.